Have pytest work with quasar

Running
```
import pytest
import torch
import ttnn
from tests.ttnn.utils_for_testing import assert_with_pcc
from models.common.utility_functions import torch_random


def test_sum(device):
    a= ttnn.ones([32,32], layout=ttnn.TILE_LAYOUT, device=device)
    b = ttnn.sum(b)
```
with
```
TT_METAL_SLOW_DISPATCH_MODE=1 pytest test_sum1.py
```
yields
```
>       if is_tg_cluster() and not device_id:

conftest.py:450: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    def is_tg_cluster():
        import ttnn
    
>       return ttnn.cluster.get_cluster_type() == ttnn.cluster.ClusterType.TG
E       ValueError: 15 is not a valid ClusterType.

conftest.py:94: ValueError
```

--- a/tt_metal/api/tt-metalium/cluster.hpp
+++ b/tt_metal/api/tt-metalium/cluster.hpp
@@ -44,6 +44,8 @@ enum class ClusterType : std::uint8_t {
     BLACKHOLE_GALAXY = 16,       // Blackhole Galaxy, all chips with mmio
     P300_X2 = 17,                // 2 P300 cards
     CUSTOM = 18,  // Custom cluster type, used boards with custom fabric mesh graph descriptor path specified
+    // NOTE:
+    // ttnn/cpp/ttnn-nanobind/cluster.cpp needs to be kept in sync with this enum class.
 };
 
 /**

--- a/ttnn/cpp/ttnn-nanobind/cluster.cpp
+++ b/ttnn/cpp/ttnn-nanobind/cluster.cpp
@@ -74,6 +74,7 @@ void py_cluster_module_types(nb::module_& mod) {
         .value("SIMULATOR_BLACKHOLE", tt::tt_metal::ClusterType::SIMULATOR_BLACKHOLE, "Simulator Blackhole")
         .value("N300_2x2", tt::tt_metal::ClusterType::N300_2x2, "2 N300 cards, ethernet connected to form 2x2")
         .value("P300", tt::tt_metal::ClusterType::P300, "Production P300")
+        .value("SIMULATOR_QUASAR", tt::tt_metal::ClusterType::SIMULATOR_QUASAR, "Simulator Quasar")
         .value("BLACKHOLE_GALAXY", tt::tt_metal::ClusterType::BLACKHOLE_GALAXY, "Blackhole Galaxy, all chips with mmio")
         .value("P300_X2", tt::tt_metal::ClusterType::P300_X2, "2 P300 cards")
         .value("CUSTOM", tt::tt_metal::ClusterType::CUSTOM, "Custom cluster");

